includes/discovery/sensors/pre-cache/siteboss.inc.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$pre_cache['siteboss_mibs'] = 'SITEBOSS-530-STD-MIB:SITEBOSS-550-STD-MIB';
+
+// caches device OID prefix from sysObjectID
+echo 'sysObjectID ';
+$oid = snmp_get($device, 'sysObjectID.0', '-OQvn', 'SNMPv2-MIB');
```

Comment:
You don't need to fetch sysObjectID.  It will always be filed in on the device so either `$device['sysObjectID']` or `$device->sysObjectID` if you have the device model.